ppx_test/errors/cstruct_duplicate_field.ml.expected

```diff
@@ -0,0 +1,2 @@
+File "cstruct_duplicate_field.ml", line 2, characters 0-77:
+Error: ppx_cstruct error: field x is present several times in this type
```

Comment:
the duplication of `error` here is a minor concern ;-) just drop the second `error`?

Reply:
it's caused by `Location.raise_errorf ~loc ("ppx_cstruct error: " ^^ fmt)` so it's global, but I agree that it should be fixed